packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.helper.ts

```diff
@@ -45,3 +46,69 @@ export const getMaximumDateFromDayClick = (clickedDate: dayjs.Dayjs, maximumDate
 
 	return clickedDate.endOf('day');
 };
+
+export const getFromDateInputState = (error: EAbsoluteTimeError | undefined, { minDate }: IAbsoluteTimeLimits): { state: EValidationState; helpText: string } | undefined => {
+	if (!error) {
+		return;
+	}
+
+	if (error === EAbsoluteTimeError.StartDateBeforeMinimumDate) {
+		const min = (isNumber(minDate) ? dayjs(minDate) : dayjs(minDate, DATETIME_INPUT_MASK)).format(DATETIME_INPUT_MASK);
```

Comment:
can we normalize the min e max value to be always a number?

Reply:
Done 👍